vendor/bat-native-confirmations/src/bat/confirmations/internal/redeem_token.cc

```diff
@@ -179,7 +180,7 @@ void RedeemToken::OnCreateConfirmation(
 
   if (response_status_code != net::HTTP_CREATED) {
     BLOG(ERROR) << "Failed to create confirmation";
-    OnRedeem(FAILED, confirmation);
+    OnRedeem(FAILED, confirmation, false);
```

Comment:
Please revert as this will break retry logic

Reply:
@SergeyZhukovsky This change is critical as it will break retry logic for desktop and mobile when a user is rewarded for viewing an ad

Reply:
@tmancey this is your change https://github.com/brave/brave-core/pull/3098/files#diff-1ca8bcdfc931634d8c307b7d978d344eR182
Feel free to fix it.

Reply:
@samartnik this is a merge issue as on `master` this is `OnRedeem(FAILED, confirmation)`

Reply:
Removed